plugins/modules/zos_mvs_raw.py

```diff
@@ -3090,6 +3103,42 @@ def get_content(formatted_name, binary=False, from_encoding=None, to_encoding=No
         return stdout
 
 
+def modify_contents(contents):
+    """Return the content of dd_input to a valid form for a JCL program.
+
+    Args:
+        contents (str or list): The string or list with the program.
+
+    Returns:
+        contents: The content in a proper multi line str.
+    """
+    if not isinstance(contents, list):
+        contents = list(contents.split("\n"))
+    contents = prepend_spaces(contents)
+    contents = "\n".join(contents)
+    return contents
+
+
+def prepend_spaces(lines):
+    """Return the array with two spaces at the beggining.
+
+    Args:
+        lines (list): The list with a line of a program.
+
+    Returns:
+        new_lines: The list in a proper two spaces and the code.
+    """
+    module = AnsibleModuleHelper(argument_spec={})
+    for index, line in enumerate(lines):
+        if len(line) > 0:
+            if len(line) > 78:
+                module.fail_json(msg="""Length of the line {0} over 80. The maximum length of the line is 80 columns including 2 spaces at the beginning.
```

Comment:
And lastly, for this error message (italics to highlight changes, the bulk of the message is really good already):

> Length of line {0} _is_ over 80. The maximum length _allowed_ is 80 columns, including 2 spaces at the beginning. If the two spaces are not _present_, the module will _add them to form syntactically correct JCL_.